osv/riscv.dpl

```diff
@@ -534,6 +536,19 @@ group:
         fmv.w.x
         fmv.d.x
         fmv.q.x
+    // atomicGrp
+        lr.w
+        sc.w
```

Comment:
What about the `.d` variant of these opcodes?  Does seL4 really only use the `.w` variant?

Reply:
Apparently so - as noted in the policy-engine pr, we don't decode those variants yet either.